drivers/hmis/app/models/hmis/wip.rb

```diff
@@ -5,6 +5,7 @@
 ###
 
 class Hmis::Wip < Hmis::HmisBase
+  acts_as_paranoid
```

Comment:
this was missing, looks like we intended it to be there since the table has a deleted_at column.

Reply:
@martha identified that this introduced a but, since the `hmis_client_projects` view doesn't check for deletion status.

@martha it looks like `hmis_households` also needs the same change, to only look at non-deleted hmis_wips. Can you make a PT story for that one?

Reply:
sure! Created https://www.pivotaltracker.com/story/show/187289783